Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 google-cloud-pubsub = "*"
-grpcio = "*"
+grpcio = "==1.13.0" # TODO: More recent versions of this cause test failures for unknown reasons, needs fixing
```

Comment:
Not ideal, but a good pragmatic approach, given our intention to deprecate this service